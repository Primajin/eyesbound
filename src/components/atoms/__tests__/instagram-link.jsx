import React from 'react';
import {create} from 'react-test-renderer';

import InstagramIcon from '../instagram-link.jsx';

describe('InstagramIcon', () => {
	it('renders correctly', () => {
		const component = create(<InstagramIcon/>).toJSON();
		expect(component).toMatchSnapshot();
	});
});
