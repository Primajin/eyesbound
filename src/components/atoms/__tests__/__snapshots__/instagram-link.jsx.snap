// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InstagramIcon renders correctly 1`] = `
<a
  css={
    Object {
      "map": undefined,
      "name": "11dms2a",
      "next": undefined,
      "styles": "
	bottom: 13px;
	display: block;
	height: 24px;
	left: 15px;
	overflow: hidden;
	position: absolute;
	width: 24px;
	z-index: 2;

	svg {
		fill: var(--background);
	}
",
      "toString": [Function],
    }
  }
  href="https://www.instagram.com/jannishell"
  rel="noreferrer"
  target="_blank"
  title="Eyesbound on Instagram"
>
  <svg
    viewBox="0 0 24 24"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M12 2.2h4.8c3.3.2 4.8 1.7 5 5a83.4 83.4 0 010 9.7c-.2 3.2-1.7 4.7-5 4.9a83 83 0 01-9.6 0c-3.3-.2-4.8-1.7-5-5a83 83 0 010-9.6c.2-3.3 1.7-4.8 5-5H12zM12 0H7C2.8.4.4 2.8 0 7a84.3 84.3 0 000 10c.3 4.3 2.7 6.7 7 7a85.6 85.6 0 0010 0c4.3-.3 6.7-2.7 7-7a84.3 84.3 0 000-10c-.3-4.3-2.7-6.7-7-7h-5zm0 5.8a6.2 6.2 0 100 12.4 6.2 6.2 0 000-12.4zM12 16a4 4 0 110-8 4 4 0 010 8zm6.4-11.8a1.4 1.4 0 100 2.8 1.4 1.4 0 000-2.8z"
    />
  </svg>
</a>
`;
